library/big-float/impl-default.lisp

```diff
@@ -760,9 +768,15 @@ returns the nth SeriesSplit, return the series evaluated to the Nth element."
         ((Tuple x (BFConst f)) (pow x (f)))
         (_ (exp (* y (ln x))))))
     (define (log b x)
-      (/ (ln x) (ln b))))
+      (/ (ln x) (ln b)))
+    (define ee (BFConst bf-ee)))
+
+  (declare bf-ee (Unit -> Big-Float))
```

Comment:
nit: this can be deleted i think

Reply:
The declare? When I take it out it fails to load